src/index.html

```diff
@@ -2,16 +2,181 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+       action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+       method="post"
+       onsubmit="onSubmit()"
+    >
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+             type="text"
+             name="surname"
+             id="surname"
+             autocomplete="off"
+             required
+          >
```

Comment:
Check your tab settings. It adds 3 tabs.
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
            required
          >
```